doc/version_history.rst

```diff
@@ -6,10 +6,27 @@
 Version History
 ###############
 
+v0.8.0
```

Comment:
Why a new version? v0.7.0 has not been released yet. Let's discuss this on Slack.

Reply:
I was hoping you would run the data-client based version of the ESS CSC on the summit with the current salobj. But if you don't plan to try that, then keeping this all part of v0.7.0 is fine with me. I have updated the version history as you asked.

Reply:
Thanks a lot for the change. I think we can test with the RC if we want to do that at some point. However, we still don't have a test sensor so that will be hard to accomplish.